Remotion/Web/Development.WebTesting/WebTestHelper.cs

```diff
@@ -190,12 +191,15 @@ public void OnSetUp ([NotNull] string testName)
     /// Creates a new browser session using the configured settings from <see cref="WebTestConfigurationFactory"/>.
     /// </summary>
     /// <param name="maximizeWindow">Specified whether the new browser session's window should be maximized.</param>
+    /// <param name="configurationOverride">Specifies additional options applied when creating the browser.</param>
     /// <returns>The new browser session.</returns>
-    public IBrowserSession CreateNewBrowserSession (bool maximizeWindow = true)
+    public IBrowserSession CreateNewBrowserSession (bool maximizeWindow = true, [CanBeNull] DriverConfigurationOverride configurationOverride = null)
     {
       using (new PerformanceTimer (s_log, string.Format ("Created new {0} browser session.", _browserConfiguration.BrowserName)))
       {
-        var browserResult = _browserConfiguration.BrowserFactory.CreateBrowser (_testInfrastructureConfiguration);
+        var mergedDriverConfiguration = MergeDriverConfiguration (_testInfrastructureConfiguration.DriverConfiguration, configurationOverride);

```

Comment:
I think there was a discussion in the original PR about not exposing DriverCOnfiguration via TestInfrastructureCOnfiguration but separate. Please dig out the discussion so we can make the final decision on the design.

Reply:
> Should we keep DriverConfiguration on TestInfrastructureConfiguration or should split those types permanently? Or, from a reverse option, should we build the DriverConfiguration in WebTestHelper and also expose it on WebTestHelper.

Our decision would be to separate the `DriverConfiguration` from the `TestInfrastructureConfiguration` and expose it through the `WebTestHelper`. The `DriverConfiguration` seems equivalent to the `BrowserConfiguration` and `TestInfrastructureConfiguration`.